daemon/api.go

```diff
@@ -1449,14 +1450,16 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	}
 
 	gecos := fmt.Sprintf("%s,%s", createData.EMail, v.OpenIDIdentifier)
-	if err := osutilAddExtraSudoUser(v.Username, v.SSHKeys, gecos); err != nil {
+	if err := osutilAddExtraUser(v.Username, v.SSHKeys, gecos, createData.Sudoer); err != nil {
 		return BadRequest("cannot create user %s: %s", v.Username, err)
 	}
 
 	var createResponseData struct {
-		Username string `json:"username"`
+		Username    string `json:"username"`
+		SshKeyCount int    `json:"ssh_key_count"`
 	}
 	createResponseData.Username = v.Username
+	createResponseData.SshKeyCount = len(v.SSHKeys)
```

Comment:
This is inspired by the behaviour when the user doesn't have ssh keys. Currently this fails, but once snapd has the magic to keep ssh keys in sync with the server it should succeed. But  console-conf should not invite the user to log in in that case!

Returning the keys might make sense, it would be nice for console-conf to present the fingerprints of the keys or something like that (not done yet).